Generalize system specific commands (#20)
boom100100/local_and_website_pdf_merger: README.md

Setup requires downloading the project and its dependencies.

### MacOs Install Script

> Only run one of the two commands. Use the standalone CLI app, or use the ATS Beater extension (it depends on the CLI app). Then, enter the sign-in password when prompted.

Author: ```suggestion
Only run one of the two commands below.
```
